src/workflows-app/SubmissionHistory.js

```diff
@@ -225,6 +225,7 @@ export const BaseSubmissionHistory = ({ namespace, workspace }, _ref) => {
                             field: 'actions',
                             headerRenderer: () => h(TextCell, {}, ['Actions']),
                             cellRenderer: ({ rowIndex }) => {
+                              const permissionToAbort = paginatedPreviousRunSets[rowIndex].user_id === termsOfService.userId;
```

Comment:
Technically this is the same way as it's used in TermsOfService... but I can swap out for a direct call to Sam rather than using `authStore`? https://github.com/DataBiosphere/terra-ui/blob/9e899203a5aa28d135f3ad75dc68b34b34c6f807/src/pages/TermsOfService.ts#L20